client/sandbox/react-nextjs/tsconfig.json

```diff
@@ -8,8 +8,8 @@
     "forceConsistentCasingInFileNames": true,
     "noEmit": true,
     "esModuleInterop": true,
-    "module": "esnext",
-    "moduleResolution": "node",
+    "module": "NodeNext",
+    "moduleResolution": "nodenext",
```

Comment:
allows importing from "sub exports", moduleResolution: "node" means compiles to support for node v10, node next raises the minimum version of node